app/api/scripts/create_eligibility_screener_csv.py

```diff
@@ -0,0 +1,153 @@
+    zip_code="Zip Code",
+    chosen_wic_clinic="Chosen WIC Clinic",
+    applicant_notes="Applicant notes",
+    submitted_datetime="Submitted date/time",
+)
+
+
+def create_eligibility_screener_csv(db_session: scoped_session, output_file_path: str) -> None:
+    # Get DB records
+    eligibility_screener_records = get_eligibility_screener_records(db_session)
+
+    # Convert records
+    csv_records = convert_eligibility_screener_records_for_csv(eligibility_screener_records)
+
+    # Create file
+    generate_csv_file(csv_records, output_file_path)
+
+    # Update DB records with timestamp
+    # Only update the timestamp if the file upload worked
+    update_db_record_timestamp(eligibility_screener_records)
+    db_session.commit()
+
+
+def get_eligibility_screener_records(db_session: scoped_session) -> list[EligibilityScreener]:
+    logger.info("Fetching Eligibility Screener records from DB")
+    eligibility_screener_records = (
+        db_session.query(EligibilityScreener)
+        .filter(EligibilityScreener.added_to_eligibility_screener_at == None)  # noqa: E711
+        .all()
+    )
+
+    record_count = len(eligibility_screener_records)
+    logger.info(
+        "Found %s eligibility screener records",
+        record_count,
+        extra={"eligibility_screener_record_count": record_count},
+    )
+
+    if record_count == 0:
+        logger.warning("No eligibility screener records found that weren't already added to a CSV")
+
+    return eligibility_screener_records
+
+
+def convert_eligibility_screener_records_for_csv(
+    records: list[EligibilityScreener],
+) -> list[EligiblityScreenerCsvRecord]:
+    logger.info("Converting eligibility screeners to CSV format")
+    out_records: list[EligiblityScreenerCsvRecord] = []
+    out_records.append(ELIGIBILITY_SCREENER_CSV_HEADER)
+
+    for record in records:
+        # TODO - do we want to convert this timestamp to a different timezone beside UTC?
+        # Create a timezone in a human-readable format: 09/06/22 05:56 PM
+        submitted_datetime = record.created_at.strftime(DATETIME_OUTPUT_FORMAT)
+
+        out_record = EligiblityScreenerCsvRecord(
+            first_name=record.first_name,
+            last_name=record.last_name,
+            phone_number=record.phone_number,
+            eligibility_categories=join_list(record.eligibility_categories),
+            has_prior_wic_enrollment=blank_for_null(record.has_prior_wic_enrollment),
+            eligibility_programs=join_list(record.eligibility_programs),
+            household_size=blank_for_null(record.household_size),
+            zip_code=record.zip_code,
+            chosen_wic_clinic="TODO",
+            applicant_notes=blank_for_null(record.applicant_notes),
+            submitted_datetime=submitted_datetime,
+        )
+        out_records.append(out_record)
+
+    return out_records
+
+
+def generate_csv_file(records: list[EligiblityScreenerCsvRecord], output_file_path: str) -> None:
```

Comment:
For the platform work (so I don't mean necessarily for this PR), I would encourage you to break out csv utilities into a separate util file).

Reply:
I agree. There are additional CSV utilities I think would be good to bring over.